src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -67,7 +67,7 @@ protected function configure(): void
                 new InputOption('limit', 'l', InputOption::VALUE_REQUIRED, 'Limit the number of received messages'),
                 new InputOption('failure-limit', 'f', InputOption::VALUE_REQUIRED, 'The number of failed messages the worker can consume'),
                 new InputOption('memory-limit', 'm', InputOption::VALUE_REQUIRED, 'The memory limit the worker can consume'),
-                new InputOption('time-limit', 't', InputOption::VALUE_REQUIRED, 'The time limit in seconds the worker can run'),
+                new InputOption('time-limit', 't', InputOption::VALUE_REQUIRED, 'The time limit in seconds the worker can handle new messages'),
```

Comment:
not sure I see why this is better sorry

Reply:
@nicolas-grekas "The time limit in seconds the worker can run" = "the worker cannot work more than this time limit"
So this sentence is wrong because the worker can actually run longer than the time limit.

Mine is right but surely not the best one, feel free to propose something if you have better ideas.

Reply:
I don't get why this tells something different. To me this is similar. Maybe it's y English...

Reply:
What about `The time limit in seconds the worker can consume new messages`?

Reply:
@nicolas-grekas the difference is
- current sentence => when time's up, drop right now what you're doing and exit
- proposed one => when time's up, take all the time you need to finish what you're doing, then exit and don't get/retrieve/consume/handle any other message

Reply:
@Guikingone thank you, I'll update